### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by Emendo12** [MikeDX/2600_gba#1]
I'd like to use this on my EZ Flash Omega flash cart for GBA. How do I make this into a useable program?

**Comment by MikeDX** [comment]
I don't think this will compile with a modern devkitpro. It might work but also the build scripts were for msdos/windows so everything may need changing.

Would be interesting to see if anyone could compile it.

**Comment by Emendo12** [comment]
So it was just an experiment? It says it's for GBA haha so I was super hyped when I found it. 

I pray that someone can get this working on real GBA hardware soon. A theme for the EZ Flash Omega based on Pogoshell, called '"Simple," has emulators for systems like the CollecoVision and NES, among others. I would LOVE to have one for Atari 2600 available to the community.

**Comment by MikeDX** [comment]
I dont have the old binaries, only this old source code. I think it will work but it was very slow. Just depends on if anyone can get it to work :)
